tensorflow/python/ops/nn_ops.py

```diff
@@ -327,6 +331,8 @@ def max_pool(value, ksize, strides, padding, data_format="NHWC", name=None):
     strides: A list of ints that has length >= 4.  The stride of the sliding
       window for each dimension of the input tensor.
     padding: A string, either `'VALID'` or `'SAME'`. The padding algorithm.
+      See the comment here for details: 
+      https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/nn.py
```

Comment:
Yes, we have fancy forwarders on tensorflow.org which figure out the right version/branch to use:

```markdown
    See the [comment here](https://tensorflow.org/code/tensorflow/python/ops/nn.py)
```

Looks like this:

See the [comment here](https://tensorflow.org/code/tensorflow/python/ops/nn.py)